Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -202,21 +207,45 @@ class MultiTrajectory {
   }
 
  public:
+  /// @anchor track_state_container_track_access
+  /// @name MultiTrajectory track state (proxy) access and manipulation
+  ///
+  /// These methods allow accessing track states, i.e. adding or retrieving a
+  /// track state proxy that points at a specific track state in the container.
+  ///
+  /// @{
+
   /// Access a read-only point on the trajectory by index.
+  /// @note Only available if the MultiTrajectory is not read-only
   /// @param istate The index to access
   /// @return Read only proxy to the stored track state
   ConstTrackStateProxy getTrackState(IndexType istate) const {
     return {*this, istate};
   }
 
   /// Access a writable point on the trajectory by index.
+  /// @note Only available if the MultiTrajectory is not read-only
   /// @param istate The index to access
   /// @return Read-write proxy to the stored track state
   template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
   TrackStateProxy getTrackState(IndexType istate) {
     return {*this, istate};
   }
 
+  /// Add a track state without providing explicit information. Which components
+  /// of the track state are initialized/allocated can be controlled via @p mask
+  /// @note Only available if the MultiTrajectory is not read-only
+  /// @param mask The bitmask that instructs which components to allocate and
+  /// which to leave invalid
```

Comment:
```suggestion
  ///       which to leave invalid
```

do you know how doxygen handles these cases?

Reply:
It should work either way I believe.